scripts/measure_trigger_eff.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+from ROOT import gROOT
+gROOT.SetBatch(True)
+import ROOT
+import json
+
+data = {}
+
+file_path = "/storage/epp2/phshgg/DVTuples__v23/5TeV_2017_32_Down_EW.root"
+
+ch = ROOT.TChain('Z/DecayTree')
+ch.Add(file_path)
+
+mup_trigger = float(ch.GetEntries("mup_L0MuonDecision_TOS == 1"))
+mum_trigger = float(ch.GetEntries("mum_L0MuonDecision_TOS == 1"))
+both_trigger = float(ch.GetEntries("mup_L0MuonDecision_TOS == 1 && mum_L0MuonDecision_TOS == 1"))
+
+
+mup_eff = both_trigger/mum_trigger
+mum_eff = both_trigger/mup_trigger
+
+mup_mum_eff = mup_eff * mum_eff
```

Comment:
This would be the correct efficiency for *both* muons to fire the trigger.
What we want is the efficiency for *either* muon to fire the trigger.

Reply:
I've had another attempt at this using:
P(either A or B) = P(A) + P(B) – P(A and B)

This gives xsec = 58 ± 3 pb as before when calculated without efficiency.

For comparison to a few extra decimal places:
- Assumption: 58.01 ± 2.76 pb
- With efficiency: 58.26 ± 2.77 pb

Reply:
:thumbsup: @lmcairns that's exactly the formula that I had in mind for the efficiency of the logical OR. (under the assumption that the two efficiencies are uncorrelated).

Reply:
The uncertainty that you quote is just the 5% relative uncertainty on the luminosity I guess?

As a next step we should estimate the statistical uncertainties from the signal yield and the efficiency.

For the signal yield the relative uncertainty is just sqrt(N)/N

For the efficiency we should use the binomial formula where the relative uncertainty is sqrt(efficiency*(1-efficiency)/N_total)

Reply:
And then try to propagate these to your cross section measurement. I'd suggest to report it (in the latex) like 
```latex
\sigma = X.XX \pm X.XX_{\rm stat} \pm X.XX_{\rm eff} \pm X.XX_{\rm lumi}
```

Reply:
The second uncertainty would eventually become the systematic uncertainty as we add more sources.

Reply:
Maybe worth merging this now though, and continuing the work with a follow up pull request